src/appkit.cs

```diff
@@ -16942,7 +16947,7 @@ public partial interface NSTextInputContext {
 		NSTextInputContext CurrentInputContext { get; }
 
 		[Export ("client")]
-		NSTextInputClient Client { get; }
+		INSTextInputClient Client { get; }
```

Comment:
mmm but wouldn't `NSTextInputClient` conform to  `INSTextInputClient`? I mean you would get an object with the same methods from `NSTextInputClient`. Am I missing something?